setups/rufus

```diff
@@ -1,3 +1,22 @@
 #!/usr/bin/env bash
 
-echo "PLACEHOLDER"
\ No newline at end of file
+Yellow='\033[0;33m'
+NC='\033[0m' # No Color
+
+git clone $CONFIG_base_remote -b $BRANCH_NAME $PROJECT_NAME
```

Comment:
I made some changes to the main script. `$PROJECT_NAME` was renamed to `$projectName`